README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello World Project
 
-> One paragraph statement about the project.
+> My fist microverse module 1 project.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+This is a simple site which displays the text "Hello Microverse".
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- Css
 
 ## Live Demo
```

Comment:
- [ ] [OPTIONAL] You can erase sections in your README file that are empty, in this way your README file will look more professional.

Reply:
:heavy_check_mark: Done.